.vscode/launch.json

```diff
@@ -18,6 +18,21 @@
                 "${workspaceRoot}/**/*"
             ],
             "envFile": "${workspaceRoot}/.env"
+        },
+        {
+            "name": "Current TS File",
```

Comment:
@CodeByEddy  FYI, you can debug the tests with VSCode. Just go to index.ts and debug it selecting this runner.